includes/Helpers/ProductTabsMetaHandler.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/**
+ * Product Tabs Meta Handler
+ *
+ * @since 1.9.1-dev.1
+ */
+namespace WooCommerceCustomProductTabsLite\Helpers;
+
+use WC_Product;
+
+class ProductTabsMetaHandler
+{
+	const PRODUCT_TABS_META_KEY = '_wc_custom_product_tabs_lite_product_tabs';
+	const LEGACY_PRODUCT_TABS_META_KEY = 'frs_woo_product_tabs';
+
+	public function addHooks()
+	{
+		add_filter('get_post_metadata', [$this, 'maybeConvertLegacyProductTabsMeta'], 10, 5);
+	}
+
+	/**
+	 * Gets product meta.
+	 *
+	 * @param WC_Product $product
+	 * @param string $context
+	 * @return mixed
+	 */
+	public function getMeta(WC_Product $product, string $context = 'edit')
+	{
+		$meta = $product->get_meta(static::PRODUCT_TABS_META_KEY, true, $context);
+
+		if (! empty($meta) && is_string($meta)) {
+			return json_decode($meta, true);
+		}
+
+		return $this->maybeMigrateLegacyMeta($product);
+	}
+
+	/**
+	 * Updates product tabs meta.
+	 *
+	 * @param WC_Product $product Passed by reference.
+	 * @param array      $meta
+	 * @return void
+	 */
+	public function updateMeta(WC_Product &$product, array|string $meta) : void
```

Comment:
I think union types are PHP 8.0 only. This plugin is PHP 7.4+ so we need to change this.

Reply:
Changed here: https://github.com/godaddy-wordpress/woocommerce-custom-product-tabs-lite/pull/39/commits/ed4bb89a998dc8d97265bd8835358967401840a6

I went with `array|mixed` in the docblock because we weren't strictly checking an array in other places. Could switch to purely `array` if we add in extra `is_array()` checks.